src/editors/data/services/lms/utils.js

```diff
@@ -0,0 +1,17 @@
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
```

Comment:
What is the benefit of creating our own module here? We can already mock the AuthenticatedHttpClient, unless I am mistaken?